app/javascript/mastodon/components/column_header.js

```diff
@@ -178,7 +178,9 @@ class ColumnHeader extends React.PureComponent {
     if (multiColumn || placeholder) {
       return component;
     } else {
-      return createPortal(component, document.getElementById('tabs-bar__portal'));
+      var e = document.getElementById('tabs-bar__portal');
+      if (e === null) return component;
```

Comment:
This branch should force the component to re-render, as otherwise, the column header will not scroll properly.